M0narc/fast_api
# users models, auth router and users requests for pydantics created

## Description
users models, auth router and users requests for pydantics created
tested in swagger.


## Related Story Ticket



## Testing Instructions

go to our swagger page and test the auth endpoint to create user, it should return 200 ok
then again, if there are unique emails and usernames it will throw an error
same thing with passwords that are not longer than eight characters.


## Screenshots


![imagen](https://github.com/M0narc/fast_api/assets/46531321/1ed92248-4512-4162-b0e1-5c260ef342c6)
